test/gtest/ucs/test_time.cc

```diff
@@ -116,6 +116,7 @@ UCS_TEST_F(test_time, timerq) {
         }
         EXPECT_EQ(0u, counter1);
         EXPECT_NEAR(test_time / interval2, counter2, 1);
+        EXPECT_LE(ucs_timerq_min_interval(&timerq), UCS_TIME_INFINITY);
```

Comment:
no, the assertions caught the bug, but i added it to the test anyway - do you mean it should be EXPECT_LT?